app/database/persisted_queries.js

```diff
@@ -0,0 +1,75 @@
+const { knex } = require("./index");
+const redis = require("../redis");
+const DEFAULT_TTL = 12 * 3600;
+
+module.exports.count = async () => {
+	return (await knex("persisted_queries").count("key", { as: "amount" }))[0]
+		.amount;
+};
```

Comment:
As it is a oneliner anyways
```suggestion
module.exports.count = async () =>
    (await knex("persisted_queries").count("key", { as: "amount" }))[0].amount;
```

Reply:
its no longer anonymous 🤔 , Aivo wanted better stacktraces